organizational/conduct.rst

```diff
@@ -0,0 +1,71 @@
+Code of Conduct
+===============
+
+Hacker Society [#hacsoc]_ is, as Tim wrote in :doc:`hacsoc`, about an open
+society:
+
+  An open society promotes freedom of thought and expression, personal
+  responsibility, individualism, cooperation, and equality of all citizens.
+
+As such, we expect that our members behave in a way that allows us to achieve
+this.  Participants in HacSoc events are expected to treat each other with
+dignity and respect for each other's perspectives, experiences, and knowledge.
+In particular, participants should recognize that people of all skill levels are
+welcome, and should always be willing to help others learn.
+
+We want Hacker Society to be a place where people can be comfortable regardless
+of gender, gender identity and expression, sexual orientation, disability,
+physical appearance, body size, race, age, or religion.  As such, we do not
+tolerate harassment.  This includes, but is not limited to:
+
+- Rude or unwelcome comments
```

Comment:
@timtadh: I agree.  The original policy I was basing this on was a bit more specific - it actually spelled out the intrinsic characteristics you talk about.  Here is the original line (note it's something of a template):

> Verbal comments that reinforce social structures of domination [related to gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, religion, [your specific concern here].

I made the line more general mostly because I was hoping it could apply to more undesirable conduct; in retrospect, I don't know what else it really would cover.  I was also unsure about the "social structures of domination" portion, since in my opinion, _any_ attack on someone based on their intrinsic characteristics is problematic, not just those that reinforce the "social structures of domination".  I might propose a simpler, but more specific wording:

> Verbal attacks or unwelcome comments about gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age, or religion